libs/agno/agno/tools/brandfetch.py

```diff
@@ -39,29 +41,32 @@ def __init__(
         client_id: Optional[str] = None,
         base_url: str = "https://api.brandfetch.io/v2",
         timeout: Optional[float] = 20.0,
+        enable_search_by_identifier: bool = True,
+        enable_search_by_brand: bool = False,
+        enable_asearch_by_identifier: bool = False,
+        enable_asearch_by_brand: bool = False,
+        all: bool = False,
```

Comment:
We only need 2 flags like before. The previous behaviour with async_tools is still valid.